
uint32 drone_id

string data_type

---

geometry_msgs/Vector3 vec3

geometry_msgs/Vector3 forward